yt_dlp/downloader/mhtml.py

```diff
@@ -199,3 +199,4 @@ def real_download(self, filename, info_dict):
         ctx['dest_stream'].write(
             b'--%b--\r\n\r\n' % frag_boundary.encode('us-ascii'))
         self._finish_frag_download(ctx)
+        return True
```

Comment:
What does this do?

Reply:
it it doesn't return True, yt-dlp assumes no actual file was downloaded and will not run postprocessing - and yes, we need some of the PP to run

Reply:
Okay then. I would assume that not throwing an exception would be enough…

Reply:
yeah, I'm, not sure what the purpose of that is either, but that is how it works